debvader/deblender.py

```diff
@@ -34,6 +34,7 @@ def __init__(
         nb_of_bands=6,
         epistemic_uncertainty_estimation=False,
         normalization_func=None,
+        denormalization_func=None,
```

Comment:
Alright, this is adding too many arguments for one simple task to be handled.
Plus I guess you have to have to ensure identical normalisation functions are used, either linear on non-linear. 
Let's try to find an alternative..

Two options come to mind : 
- use a class e.g. `LinearCosmos` and `NonLinearCosmos` with methods `normalize()` and `denormalize()`
- use more specific keywords like `normalization_type="linear"` and `normalization_survey="cosmos"`, and from these, go fetch the right function accordingly.

Reply:
I agree with the point there should be only one function. But I want to give the user the flexibility to choose easily define the normalization function. 
I just push changes to the PR, does it look ok?